src/switch_jitterbuffer.c

```diff
@@ -332,6 +361,29 @@ static inline void hide_nodes(switch_jb_t *jb)
 	switch_mutex_unlock(jb->list_mutex);
 }
 
+static inline switch_bool_t packet_vad(switch_jb_t *jb, switch_rtp_packet_t *packet, switch_size_t len) {
+	void* payload;
+	uint16_t payload_len = len;
+
+	if (packet->ebody) {
+		payload = packet->ebody;
+	} else {
+		payload = packet->body;
+	}
+	if (payload && payload_len > 0) {
+		switch_bool_t ret;
+		switch_bool_t *ret_p = &ret;
+		switch_codec_control_type_t ret_t;
+		switch_core_media_codec_control(jb->session, SWITCH_MEDIA_TYPE_AUDIO,
```

Comment:
The idea with the getters is that when JB is created or reset we'd get the engine, then the codec and we save them so we don't do it all the time, for each packet.